src/app/RecordingList/ActiveRecordingsList.tsx

```diff
@@ -219,6 +219,16 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
           id={`active-ex-expand-${props.index}`}
           isHidden={!isExpanded}
         >
+          <Text>Recording Options:</Text>
+          <Text>
+            toDisk = { String(props.recording.toDisk) } &emsp; 
+            maxAge = {props.recording.maxAge / 1000}s &emsp; 
+            maxSize = { props.recording.maxSize }B
+          </Text>
+          <br></br>
+          <hr></hr>
+          <br></br>
```

Comment:
`hr` and `br` are empty elements in HTML, so these should simply be `<br>` and `<hr>` with no `</br>` closing tags.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
Im trying to understand how the Text component works, and it seemes like it should work but for some reason it gives an error when I don't close hr and br.

Reply:
Hmm. Maybe it's a quirk of TSX syntax, or the way that it's compiled. No problem then.